packages/core/src/components/TextField.tsx

```diff
@@ -270,7 +276,7 @@ class TextField extends React.Component<Props, State> {
     } else {
       activeColor = error ? colors.error : colors.primary;
       placeholderColor = borderColor = colors.light;
-      underlineColor = colors.light;
+      underlineColor = underlineColorProp || colors.light;
```

Comment:
```suggestion
      underlineColor = underlineColorProp;
```
Could we make this the defaultValue?